test/CmakeRegenTest.cpp

```diff
@@ -312,8 +312,7 @@ TEST(RegenerateCmakeAddTarget_Interface) {
   };
 
   const std::string expectedOutput(
-      "add_library(${PROJECT_NAME} INTERFACE\n"
-      "  EXCLUDE_FROM_ALL\n"
+      "add_library(${PROJECT_NAME} EXCLUDE_FROM_ALL INTERFACE\n"
```

Comment:
Yeah... this is wrong. I want INTERFACE to come first as the defining type of the library.

Reply:
As mentioned earlier there is no strict order necessary and this output is depending on how the user has defined it before already.